docs/docs/how-to/creating-packages.mdx

```diff
@@ -84,7 +88,12 @@ A `k8s.sh` script could look like this:
 
 k8sMainRootFilePath=$( cd "$(dirname "${BASH_SOURCE[0]}")" ; pwd -P )
 
-if [ "$1" == "up" ]; then
+if [ "$1" == "init" ]; then
+    echo "execute other setup scripts or commands here..."
```

Comment:
So this was meant to distinguish the `init` code block from the `up` code block. Maybe it should just be a comment rather than an echo command?

Reply:
ok cool, yeah i think a comment might be better instead of an echo. Just wasnt sure what text was meant for. Maybe the comment can be a bit more descriptive to indicate its used for any additional initialization scripts for the setup

Reply:
Actually, lets leave it as the comment you have :+1: